actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+*   Prefer `remove_method` over `undef_method` when reloading routes
+
+    When `undef_method` is used it prevents access to other implementations of that
+    url helper in the ancestor chain so use `remove_method` instead to restores access.
+
+    *Andrew White*
+
+*   Add the `resolve` method to the routing DSL
+
+    This new method allows customization of the polymorphic mapping of models:
+
+    ``` ruby
+    resource :basket
+    direct(class: "Basket") { [:basket] }
+    ```
```

Comment:
Is this a typo, it says "Add the `resolve`" method, but then the example uses `direct` not `resolve`.  Not sure if the example is valid as it is and is demonstrating `direct`, or if it's a typo and the example is demonstrating `resolve`.  Trying to figure out these new features.